Assets/_GameJamCat/CatGenerator.asset

```diff
@@ -0,0 +1,43 @@
+%YAML 1.1
```

Comment:
Can you place this inside the scripts folder 
preferably make a new folder inside script called "ScriptableObjects"